deploy/olm-catalog/ibm-mongodb-operator/0.0.1/ibm-mongodb-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -42,37 +42,15 @@ spec:
   customresourcedefinitions:
     owned:
     - description: MongoDB is the Schema for the mongodbs API
-      displayName: MongoDB
       kind: MongoDB
       name: mongodbs.operator.ibm.com
-      resources:
```

Comment:
Same as `displayName:`, we need to keep this part

Reply:
updated